src/libraries/Directory.Build.targets

```diff
@@ -35,23 +35,16 @@
     <!-- Reset these properties back to blank, since they are defaulted by Microsoft.NET.Sdk -->
     <WarningsAsErrors Condition="'$(WarningsAsErrors)' == 'NU1605'" />
 
-    <IsRuntimeAssembly Condition="'$(IsRuntimeAssembly)' == '' and
-                                  '$(IsReferenceAssembly)' != 'true' and
-                                  '$(IsGeneratorProject)' != 'true' and
-                                  '$(IsTestProject)' != 'true' and
-                                  '$(IsTestSupportProject)' != 'true' and
-                                  '$(UsingMicrosoftNoTargetsSdk)' != 'true' and
-                                  '$(UsingMicrosoftTraversalSdk)' != 'true'">true</IsRuntimeAssembly>
     <IsRuntimeAndReferenceAssembly Condition="'$(IsRuntimeAndReferenceAssembly)' == '' and
-                                              '$(IsRuntimeAssembly)' == 'true' and
+                                              '$(IsSourceProject)' == 'true' and
                                               Exists('$(LibrariesProjectRoot)$(MSBuildProjectName)') and
-                                              !Exists('$(LibrariesProjectRoot)$(MSBuildProjectName)/ref') and
+                                              !Exists('$(LibrariesProjectRoot)$(MSBuildProjectName)$([System.IO.Path]::DirectorySeparatorChar)ref') and
```

Comment:
does the forward slash actually cause a problem here?

Reply:
It shouldn't for an Exists function. In other cases it does, i.e. when doing a Contains property function.